translated: quickstart folder

hokosugi opened the pull request:
qiuckstartフォルダの中で未翻訳の4ファイルを翻訳しました。
疑問点、不明点はないですが、`hello10mins.md` ファイルの原文コメントアウトが原文に複数あり、全体をコメントアウトできなかったので、都度 `<!--` に入れて全体をコメントアウトいたしました。
原文の一行前にコメントを入れて後から分かるようにしましたが、これでよいでしょうか？

```
<!--575,580,652,666,686,692,725,754,808,877行はコメントアウトが入れ子にできないために追加しました-->
```

hokosugi committed:
translated: quickstart folder

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
 
-This is a quick tutorial to deploy a "Hello World" dapp to the Internet Computer (IC) in 10 minutes or less. Deployment of the dapp only requires basic knowledge of using a terminal. 
+"Hello World" Dapp を Internet Computer (IC) に 10 分以内でデプロイするためのクイックチュートリアルです。アプリのデプロイに必要な知識はターミナルの使い方のみです。
 <!-- Code editing knowledge is not necessary. -->
```
git の diffが適切に動かなくなりますので、原文のコメントアウトはコピーする必要はないかなと思いました。
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+"Hello World" Dapp を Internet Computer (IC) に 10 分以内でデプロイするためのクイックチュートリアルです。アプリのデプロイに必要な知識はターミナルの使い方のみです。
 <!-- Code editing knowledge is not necessary. -->
 
+始める前に、オンチェーンで動作しているこの Dapp のバージョンを見てみましょう：<https://6lqbm-ryaaa-aaaai-qibsa-cai.ic0.app/>
```
日本語に若干違和感を感じたのでsuggestionです
```suggestion
始める前に、この Dapp がオンチェーンで動作しているのを見てみましょう：<https://6lqbm-ryaaa-aaaai-qibsa-cai.ic0.app/>
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+
+1.  Canister SDK をインストールする
+2.  ローカルで Dapp をビルドしてデプロイする
+3.  Dapp の動力（ガス）となる無料 Cycle を収集する
```
collectを収集と訳すと[「趣味、研究目的として・・」](https://ejje.weblio.jp/content/collect)という意味が含まれる感じがしまうので、受け取るではいかがでしょうか
```suggestion
3.  Dapp の動力（ガス）となる無料 Cycle を受け取る
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+4.  Cycle ウォレットを作成し、そこから他の Dapp に Cycle を転送する
+5.  Dapp をオンチェーンにデプロイする
+
+このシンプルな `Hello` Dapp は、2つの [Çanister スマートコントラクト](https://wiki.internetcomputer.org/wiki/Glossary#C)（バックエンド用とフロントエンド用）で構成されています。このアプリはテキストを入力として受け取り、greeding を返します。例えば、Canister SDK（Canisterのインストール方法は後述）を使ってコマンドラインで `greet` メソッドに `Everyone` というテキスト引数を与えて呼び出すと、ターミナルに `Hello, Everyone!` と返されます：
```
なぜかセディーユになっているのと、typoです。
Greetingはこのページで頻出なので英単語として残しておきたい気持ちがありつつ、一般用語なので訳もあると良いと思いましたのでかっこがきにするのはどうでしょうか（suggestionです）
```suggestion
このシンプルな `Hello` Dapp は、2つの [Canister スマートコントラクト](https://wiki.internetcomputer.org/wiki/Glossary#C)（バックエンド用とフロントエンド用）で構成されています。このアプリはテキストを入力として受け取り、挨拶（ greeting ）を返します。例えば、Canister SDK（Canisterのインストール方法は後述）を使ってコマンドラインで `greet` メソッドに `Everyone` というテキスト引数を与えて呼び出すと、ターミナルに `Hello, Everyone!` と返されます：
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+
+このチュートリアルは `16.*.*` よりも高いバージョンの node.js で最適に動作します。
+
+<!-- Besides installing node.js, users need to also install: 
+* Node Package Manager (NPM). (This comes packaged with Node, but you may want to upgrade with `npm i -g npm`) 
+* Node Version Manager (NVM), see [installing NVM](https://github.com/nvm-sh/nvm#installing-and-updating).
+* Once you have NVM, install the latest stable build with `nvm install --lts` -->
```
上と同じです
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+
+## 2. プロジェクトを生成する（１分）
+
+<!-- After the SDK is installed, create the default "Hello World" project with two canisters (backend and frontend).
+
+The SDK comes with a starter default project that has a backend in Motoko and frontend code in HTML, CSS, and JS. Developers can use this default project to start their own dapps. In this tutorial, we will build and deploy this bundled project, so there is no need to download any other dapp code. -->
+
+<!-- The `dfx new hello` command uses the template code to create a new project directory named `hello`, template files, and a new `hello` Git repository for your project. You can create many projects with many names. -->
+
+<!-- This is roughly analogous in Web2 to Rail’s `rails new`, React.js’s `create-react-app`, or Rust’s `cargo new`. -->
+
+<!-- To create a new project for your first application: -->
+
+<!-- ### 2.1 Open a Terminal and Create a new project named "hello" -->
```
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+
+![cd new](_attachments/cd-hello.png)
+
+<!-- ### 2.3 Understanding your dapp project -->
```
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
````diff
@@ -1,8 +1,583 @@
+
+![dfx new](_attachments/dfx-new-hello-2.png)
+
+<!-- ### 2.2 Move to your project directory
+
+``` bash
+cd hello
+```
+
+Your directory should look like this: -->
````
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+
+### 実行環境の起動する
+
+<!-- Navigate to the root directory for your project, if necessary. In this tutorial, you should be in the folder `hello` because that is the name of the project created in section 2 above.
+
+Start the local canister execution environment in Terminal A: -->
```
```suggestion

```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+これは Canister 実行環境のみなので、このデプロイはメインネットへのデプロイよりも手順が少ないですが、[Cycle](../../concepts/tokens-cycles.md) が必要です。
+:::
+
+<!-- To deploy your first dapp locally:
+
+#### 3.2.1 Check that you are still in the root directory for your project, if needed.
+
+Ensure that node modules are available in your project directory, if needed, by running the following command (it does not hurt to run this many times): -->
```
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
````diff
@@ -1,8 +1,583 @@
+npm start
+```
+
+<!-- ### 3.4.2 Test the dapp locally in the browser
+
+To see your dapp running locally in the browser on http://localhost:8080. -->
````
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -1,8 +1,583 @@
+
+![hello dapp and cycles wallet](_attachments/3-canisters-hello-dapp.png)
+
+<!-- ### 4.2 Check the connection to the Internet Computer (terminal B) -->
```
```suggestion
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -74,7 +649,7 @@ This tutorial works best with a node.js version higher than `16.*.*`.
 * Node Version Manager (NVM), see [installing NVM](https://github.com/nvm-sh/nvm#installing-and-updating).
 * Once you have NVM, install the latest stable build with `nvm install --lts` -->
-
+<!--
```
細かいですが、改行は残していただくと将来的にコンフリクトが少なくなります
```suggestion

<!--
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -88,7 +663,7 @@ The SDK comes with a starter default project that has a backend in Motoko and fr
 
 <!-- ### 2.1 Open a Terminal and Create a new project named "hello" -->
-
+<!--
```
```suggestion

<!--
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -108,13 +683,13 @@ cd hello
 
 Your directory should look like this: -->
-
+<!--
```
```suggestion

<!--
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -108,13 +683,13 @@ cd hello
 
 <!-- ### 2.3 Understanding your dapp project -->
-
+<!--
```
```suggestion

<!--
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -147,7 +722,7 @@ To distinguish the two terminals in this tutorial, terminal A has a dark blue ba
 
 Start the local canister execution environment in Terminal A: -->
-
+<!--
```
```suggestion

<!--
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -176,7 +751,7 @@ Since this is only a canister execution environment, this deployment has fewer s
 
 Ensure that node modules are available in your project directory, if needed, by running the following command (it does not hurt to run this many times): -->
-
+<!--
```
```suggestion

<!--
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -230,7 +805,7 @@ npm start
 
 To see your dapp running locally in the browser on http://localhost:8080. -->
-
+<!--
```
```suggestion

<!--
```

hoosan commented on docs/developer-docs/quickstart/hello10mins.md:
```diff
@@ -299,7 +874,7 @@ By the end of this section, you will now have three canisters:
 
 <!-- ### 4.2 Check the connection to the Internet Computer (terminal B) -->
-
+<!--
```
```suggestion

<!--
```

hoosan reviewed (changes requested):
こちら改変部が多く移行が大変に感じていた箇所でして、迅速な翻訳対応大変ありがとうございます🙇‍♂️
コメントアウト、多いと厄介ですね。。
方針としては都度<!--に入れていただくので問題なく、私も他の部分で同様の対応をしております。
gitのdiffで追加された部分は分かりますのでコメントはなくても問題ございません。

その他翻訳に関していくつかコメントしましたのでご確認お願いします🙏

hokosugi committed:
Apply suggestions from code review

ありがとうございます。
日本語のフォントがいつもの感じではないので、そんなものかと思っており、セディーユとは気が付きませんでした。今後、本文中のコメントアウトは削除、コメントアウトが複数ある場合は改行を入れていきます。

Co-authored-by: hoosan <40290137+hoosan@users.noreply.github.com>

hokosugi committed:
Apply suggestions from code review2

コミットし忘れです。

Co-authored-by: hoosan <40290137+hoosan@users.noreply.github.com>
